app/partials/sfox/verify.jade

```diff
@@ -92,18 +92,31 @@
 
   .width-60
     label.f-14(translate="{{state.idType + '_ID'}}.UPLOAD_TITLE")
-    div.file-upload(ngf-drop ngf-select ng-model="state.file" ngf-drag-over-class="'dragover'" ngf-pattern="'image/*'")
-      span.type-h5.em-300.mtvl(ng-hide="state.file") Drag and Drop Document (PDF, JPG, or PNG)
-      span.type-h5.em-300.mtvl(ng-show="state.file")
-        b {{state.file.name}}
+    div.file-upload(ng-hide="state.webcam.streaming" ngf-drop ngf-select ng-model="state.file" ngf-drag-over-class="'dragover'" ngf-pattern="'image/*'")
```

Comment:
It might be worth moving web upload stuff into its own directive. We may need this in other places.